app/code/Magento/CatalogSearch/Model/Search/Catalog.php

```diff
@@ -33,58 +35,53 @@ class Catalog extends \Magento\Framework\DataObject
      *
      * @var \Magento\Backend\Helper\Data
      */
-    protected $_adminhtmlData = null;
+    protected $_adminHtmlData = null;
 
     /**
-     * @param \Magento\Backend\Helper\Data $adminhtmlData
+     * @param \Magento\Backend\Helper\Data $adminHtmlData
      * @param \Magento\Framework\Stdlib\StringUtils $string
      * @param QueryFactory $queryFactory
      */
     public function __construct(
-        \Magento\Backend\Helper\Data $adminhtmlData,
+        \Magento\Backend\Helper\Data $adminHtmlData,
```

Comment:
This change is considered to be backwards incompatible, please revert to maximize backwards compatibility